elyra/metadata/schemaspaces.py

```diff
@@ -1,17 +1,15 @@
+#!/usr/bin/env python3
+# ********************************************************************************************
```

Comment:
we need to prepend license in addition to the license that already exists. Please do not delete any of the existing license